### PR TITLE
improve docs for borg with-lock

### PR DESCRIPTION
**Pull request by kmille**
Improve docs for borg with-lock as discussed in https://github.com/borgbackup/borg/issues/8022

### CODE REVIEW

File: src/borg/archiver/lock_cmds.py [hunk 1]
```diff
@@ -73,7 +73,11 @@ def build_parser_locks(self, subparsers, common_parser, mid_common_parser):
 
         with_lock_epilog = process_epilog(
             """
-        This command runs a user-specified command while the repository lock is held.
+        This command runs a user-specified command while locking the repository. For example:
+
+        ::
+
+        $ borg with-lock /mnt/borgrepo rsync -av /mnt/borgrepo /somewhere/else
```
Comment (ThomasWaldmann) [inline src/borg/archiver/lock_cmds.py:80]: please indent line 80 by 4 spaces.

also, guess it should be `/somwhere/else/borgrepo`.

**Comment by kmille** [comment]
Fixed

**Comment by kmille** [comment]
Okay. Do you get a notification if I push something?

**Review by ThomasWaldmann — APPROVED** [review]
LGTM!

**Comment by kmille** [comment]
Should an updated `docs/usage/with-lock.rst.inc` also be part of the commit/MR?

**Comment by ThomasWaldmann** [comment]
No, see the comment in the first line of that file.

I usually run `build_usage` and `build_man` at release time, so that stuff does not need to clutter each PR.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Comment by ThomasWaldmann** [comment]
Can you backport to 1.2-maint and 1.4-maint?

The syntax is the old one, like you had it before the correction needed for borg2 / master.

**Pull request merged**